src/NServiceBus.Core.Tests/Encryption/skip_props_and_fields_that_cannot_be_set.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace NServiceBus.Core.Tests.Encryption
+{
+    using System.Collections.Generic;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class skip_props_and_fields_that_cannot_be_set : WireEncryptedStringContext
```

Comment:
Nitpicking but if we go with "BDD" style syntaxes for unit tests I would name this:

```
class When_message_contains_props_and_fields_that_cannot_be_set
{
         [Test]
          public void Should_ignore_those_properties_and_fields()...
}
```

or something along those lines